### PR TITLE
Code Style Fixes

### PR DESCRIPTION
**Pull request by vkmrishad**
- Code Style Fixes

### CODE REVIEW

File: kong/plugins/oauth2/access.lua [hunk 1]
```diff
@@ -532,26 +532,26 @@ end
...
+    return false, {status = 401, message = {[ERROR] = "invalid_token", error_description = "The access token is invalid or has expired"}, headers = {["WWW-Authenticate"] = 'Bearer realm = "service" error = "invalid_token" error_description = "The access token is invalid or has expired"'}}
   end
 
   -- Check expiration date
   if token.expires_in > 0 then -- zero means the token never expires
     local now = timestamp.get_utc()
     if now - token.created_at > (token.expires_in * 1000) then
-      return false, {status = 401, message = {[ERROR] = "invalid_token", error_description = "The access token is invalid or has expired"}, headers = {["WWW-Authenticate"] = 'Bearer realm="service" error="invalid_token" error_description="The access token is invalid or has expired"'}}
+      return false, {status = 401, message = {[ERROR] = "invalid_token", error_description = "The access token is invalid or has expired"}, headers = {["WWW-Authenticate"] = 'Bearer realm = "service" error = "invalid_token" error_description = "The access token is invalid or has expired"'}}
```
Comment (thibaultcha) [inline kong/plugins/oauth2/access.lua]: Those changes in the user-facing error messages are undesired (and unnecessary). Could you remove them from your patch? Thanks.

File: kong/plugins/ldap-auth/access.lua [hunk 2]
```diff
@@ -177,7 +177,7 @@ local function do_authentication(conf)
       scheme = upper(scheme)
     end
 
-    ngx.header["WWW-Authenticate"] = scheme .. ' realm="kong"'
+    ngx.header["WWW-Authenticate"] = scheme .. ' realm = "kong"'
```
Comment (thibaultcha) [inline kong/plugins/ldap-auth/access.lua]: ditto

**Comment by vkmrishad** [comment]
@thibaultcha ,
Reverted file changes.
Thank you

**Comment by vkmrishad** [comment]
Thank you

**Comment by vkmrishad** [comment]
@thibaultcha 
Haven't heard about Hacktoberfest swag details

**Pull request merged**